CMakeLists.txt

```diff
@@ -67,13 +67,15 @@ endif ()
   set(CIRCT_BUILT_STANDALONE 1)
   set(BACKEND_PACKAGE_STRING "LLVM ${LLVM_PACKAGE_VERSION}")
 else()
-  set(MLIR_MAIN_SRC_DIR ${LLVM_MAIN_SRC_DIR}/../mlir/include ) # --src-root
-  set(MLIR_INCLUDE_DIR ${LLVM_MAIN_SRC_DIR}/../mlir/include ) # --includedir
+  set(MLIR_MAIN_SRC_DIR ${LLVM_MAIN_SRC_DIR}/../mlir ) # --src-root
```

Comment:
ThIs change may break some paths in the esi integration tests.

Reply:
I was wondering this... I'll check the integration tests before landing this and make sure I don't break them.

Reply:
I don't think this change broke the ESI cosim tests, at least on my local setup. By the way, I found that I needed to install pycapnp to make [these](https://github.com/llvm/circt/blob/main/integration_test/ESI/cosim/cosim.py#L3) work. Should that be documented/installed automatically by the get-capnp.sh script?

Reply:
Also ran the integration tests on this branch using the GitHub actions: https://github.com/llvm/circt/runs/2063893359

Reply:
Thanks!

Reply:
Pycapnp fix https://github.com/llvm/circt/commit/745d65e6ccd5047176fc12f7a7737b95d5e7b11f